Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -405,10 +406,14 @@ int main(int argc, char** argv, char** const envp) {
   FEXCore::Profiler::Init();
   FEXCore::Telemetry::Initialize();
 
-  RootFSRedirect(&Program.ProgramPath, LDPath());
-  InterpreterHandler(&Program.ProgramPath, LDPath(), &Args);
+  if (Program.ProgramPath.starts_with(LDPath())) {
+    // From this point on, ProgramPath needs to not have the LDPath prefixed on to it.
```

Comment:
It's probably better to move towards RootFS long-term since that's what every other place referring to that setting calls it.

BTW, is ProgramPath used for anything else (other than `/proc/self/exe` and `argv[0]`) after this point? Is removing the prefix the right choice for all other uses as well?

Reply:
> It's probably better to move towards RootFS long-term since that's what every other place referring to that setting calls it.

A follow-up PR can be done to rename the variable names from `LDPath` to `RootFS`, there's a couple of places that can be hit at once.
Additional followup PR should be to rename `APP_FILENAME` to `APP_FILEPATH` because it's not the filename but the fully qualified guest path. But that should be independent.

> BTW, is ProgramPath used for anything else (other than `/proc/self/exe` and `argv[0]`) after this point? Is removing the prefix the right choice for all other uses as well?

* It also gets passed to the ELFCodeLoader, but it correctly handles the rootfs prefixing as necessary.
* It gets passed to seccomp, but only for a log.
* It gets passed to gdbserver, but only for removing the  executable from the segment map for some reason?
  * Probably something to check on later, won't really affect this.